cmake/packaging.cmake

```diff
@@ -149,6 +149,9 @@ function(generateInstallTargets)
     file(COPY "${CMAKE_SOURCE_DIR}/packs" DESTINATION "${CMAKE_BINARY_DIR}/package/linux")
     install(DIRECTORY "${CMAKE_BINARY_DIR}/package/linux/packs" DESTINATION share/osquery)
 
+    file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/certs.pem" DESTINATION "${CMAKE_BINARY_DIR}/package/linux")
```

Comment:
There's no need to copy the file in the binary folder here. That folder exists only for files that has to be processed in  some way before being consumed by the install step.
Just install the file directly from source.

Reply:
That makes sense. I was trying to follow the conventions above where `${CMAKE_SOURCE_DIR}/tools/deployment/osqueryd.service` and similar files are moved into a directory within the binary path.

Reply:
Yep sorry, that file needs a cleanup.